provider-service/base/pkg/config/config.go

```diff
@@ -27,28 +25,25 @@ type Server struct {
 	Port int    `mapstructure:"port"`
 }
 
-func LoadConfig(ctx context.Context) (*Config, error) {
-	logger := common.LoggerFromContext(ctx)
```

Comment:
I moved the logging out of the `LoadConfig` function to `main.go` (for each provider). Partly to keep this as a business-logic-only function and partly because we previously had some config-related logging in `main.go` and some in here (inconsistent).